go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/cosmos/cosmos-sdk v0.45.1
 	github.com/cosmos/go-bip39 v1.0.0
 	github.com/cosmos/ibc-go/v2 v2.2.0
+	github.com/desmos-labs/desmos/v2 v2.3.1
```

Comment:
This shouldn't be here. Watch out you're not importing anything from `github.com/desmos-labs/desmos/v2` in any of the Go file (a simply search should find where you're using this).